templates/landing/homepage.html

```diff
@@ -0,0 +1,62 @@
+{% extends 'landing/app_layout.html' %}
+{% load static %}
+{% block content %}
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
I do not understand

There is html and body landing/app_layout.html'
Am i wrong here?

Reply:
@Yarboa, 
If I understand your question correctly, Yes there are two HTML files.
1. `app_layout.html` that will wrap each of the platform pages
 (contains only the navbar and main container for the wrapped content
2. `homepage.html` which extends the `app_layout.html` and contains the home page content.

LMK if you meant otherwise

Reply:
so why do you need two heads.
MHO one could organize the main menu and page format,
Once calling to new html page, only the relevant body content is changing with bootstrap components, is there some thing prevent you from doing this?

Reply:
@Yarboa done